great_expectations/experimental/datasources/interfaces.py

```diff
@@ -408,6 +406,19 @@ def execution_engine_type(self) -> Type[ExecutionEngine]:
     # End Abstract Methods
 
 
+@dataclasses.dataclass(frozen=True)
+class HeadData:
+    """
+    An immutable wrapper around pd.DataFrame for .head() methods which are intended to be used for visual inspection of
```

Comment:
nit: This line is over 100 chars long. Could you shorten it?

Reply:
I made it shorter, although right now we state 120 char limit in the style guide (I had to look this up). Something to debate.